### PR TITLE
Remove ChangeTierRequest

### PR DESCRIPTION
**Pull request by rajivnathan**
Related Issue: https://issues.redhat.com/browse/CRT-1473

Related PR: https://github.com/codeready-toolchain/api/pull/312

### CODE REVIEW

File: PROJECT [hunk 1]
```diff
@@ -14,14 +14,6 @@ resources:
   kind: BannedUser
   path: ../github.com/codeready-toolchain/api/api/v1alpha1
   version: v1alpha1
-- api:
-    crdVersion: v1
```
Comment (ranakan19) [inline PROJECT:18]: is this file auto-generated?

**Review by ranakan19 — APPROVED** [review]
Looks Good! 
asked a question, not a blocker though

File: PROJECT (hunk 1 above)
Comment (rajivnathan) [reply PROJECT:18]: No, it's not auto-generated. It's a manual change when adding/removing a CRD. It's mentioned in the api repo's PR template: https://github.com/codeready-toolchain/api/blob/master/.github/pull_request_template.md

**Pull request merged**